http.md

```diff
@@ -98,3 +101,25 @@ const buffer = Buffer.concat(chunks);
 
 const image = await Jimp.read(buffer);
 ```
+
+This can also be used to serve custom 404 HTML pages from within an HTTP framework, if desired.
+
+To return a dynamic response, your application needs to handle the requested path and return the desired response. The details of how to do this depend on the framework you are using. For example, using Express you can add a default handler:
+
+```javascript
+import { http } from "@ampt/sdk"
+import express from "express"
+
+const app = express()
+
+app.use('/api', ...)
+
+app.use((req, res) => {
+  // a basic 404 response
+  res.status(404).send('Sorry that page was not found')
+
+  // if you want to return your 404 page
```

Comment:
This doesn't work, BTW. I get this error when I try it:
```TypeError: The "chunk" argument must be of type string or an instance of Buffer or Uint8Array. Received an instance of ReadableStream```

Reply:
Nice I think it'll need to be converted. I'll update.